google/data_source_google_service_account_id_token_test.go

```diff
@@ -0,0 +1,102 @@
+package google
+
+import (
+	"testing"
+
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+)
+
+const targetAudience = "https://foo.bar/"
+const fakeIdToken = "eyJhbGciOiJSUzI1NiIsIm..."
+
+func testAccCheckServiceAccountIdTokenValue(name, value string) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		ms := s.RootModule()
+		rs, ok := ms.Outputs[name]
```

Comment:
The first test fails on this  line:
```
------- Stdout: -------
=== RUN   TestAccDataSourceGoogleServiceAccountIdToken_basic
=== PAUSE TestAccDataSourceGoogleServiceAccountIdToken_basic
=== CONT  TestAccDataSourceGoogleServiceAccountIdToken_basic
TestAccDataSourceGoogleServiceAccountIdToken_basic: testing.go:674: Step 0 error: Check failed: Check 2/2 error: Not found: google_service_account_id_token.default
--- FAIL: TestAccDataSourceGoogleServiceAccountIdToken_basic (0.72s)
FAIL
```

I think you want to use `ms.Resources` here and then pass it in `resourceName` in the test.